example/modules/ROOT/pages/intro.adoc

```diff
@@ -1,3 +1,4 @@
 # This is an example
 
 Here is a test example.
```

Comment:
This is some review
comment https://github.com/couchbaselabs/docs-style-guide/blob/main/ValeStyles/Couchbase/ComplexContractions.yml

```suggestion
wibble wibble wibble
this is some testing
```